content/posts/2025/01/teaching-llms-report.md

```diff
@@ -0,0 +1,98 @@
+A common theme among participants' descriptions was a need for "myth-busting", to expose and correct common misconceptions among learners about how these tools work, their limitations, and the differences between LLM assistants and other tools (e.g. a search engine).
+Describing their motivation to teach LLM assistants, several participants observed that their _learners are already making extensive use of the tools in their work/studies/day-to-day life_. 
+In that context, they felt that it is essential to acknowledge the existence of LLM assistants, discuss their potential uses and hazards, and try to ensure that learners have a good understanding of how to make safe and appropriate use of them.
+
+
+## Advantages and disadvantages of teaching LLM assistants in The Carpentries 
+Participants returned to that latter point when discussing the potential advantages and disadvantages of teaching LLM assistants in Data Carpentry, Library Carpentry, and Software Carpentry workshops: in many cases, learners will already be using these tools, or will be curious about them if not. 
+Although many participants in the discussion sessions had strong misgivings about these technologies, there was a general sense that we cannot omit them from workshops altogether.
+Some pointed to evidence and personal experience suggesting that learners who already have some working mental model of programming are able to use LLMs more effectively to augment their approach, but those who do not have the skills, vocabulary, or confidence are less likely to be able to use the "assistance" offered by such tools.
+Instead, just as Carpentries workshops aim to instil good (enough) practices in software development and data organisation/handling, the majority of session participants saw Carpentries workshops as an opportunity for Instructors to teach good practices and help learners develop a better, safer, more useful understanding of LLMs.
+
+Furthermore, some participants asserted that usage of LLM assistants is increasingly widespread in the workplace and will likely become an expected skill in the coming years. 
+They felt that we would be doing learners a disservice by not teaching them the skills and knowledge to use these tools as part of their routine approach to software development and data analysis.
+
+However, participants also identified several potential downsides to formally introducing content on LLM assistants into the curriculum for Carpentries workshops. 
+First, that LLM tools may be unavailable or forbidden at some institutions, for example where usage is restricted to a specific platform on data privacy/regulatory grounds.
+This could make it difficult to develop content, examples, etc that are specific to a particular tool with confidence that it will be usable at workshops in all locations.
```

Comment:
```suggestion
This could make it difficult to develop content and examples that are specific to a particular tool with confidence that they will be usable at workshops in all locations.
```